### DISCUSSION TITLE
Image Handling

### DISCUSSION THREAD

**Discussion by varphi-online** [TimoKats/mdrss#33] [Q&A]
When it comes to locally linked images, due to the nature of file serving, would we want to duplicate any images into some user-specified directory so the server can access them? If not, what is the preferrable method of linking those images to be served alongside the feed?

Also, I would love to be able to add an optional feed image via user config as well, is that within the scope of this project at this time?

**Comment by TimoKats** [comment]
Hi Varphi. Yeah I know I put that in TODO but it's not a necessary feature I think. As for next steps, I haven't worked on this project in a bit, but my guess is that it mostly needs refactoring to become v1.0.0.

From that point forward, perhaps RSS encryption might be an interesting route to take? That's at least something that I would use.  But don't start developing that yet. As mentioned, this project has quite a bit of technical debt to pay off💸
